### PR TITLE
Added Algolia published and unpublished date filtering

### PR DESCRIPTION
**Pull request by Jlaird**

### CODE REVIEW

File: sites/securityinfowatch.com/config/site.js [hunk 1]
```diff
@@ -69,6 +69,6 @@ module.exports = {
   },
   algolia: {
     applicationId: '80YJALCHBD',
-    searchApiKey: '54e907df9d8e8e66dbaf63ac1938c740',
+    searchApiKey: 'N2NkMTFjMjI3ZTFhZTVjNGJmNWU3ODE5ZTM5MDgyMzQzMWFmMWMxNjQyY2NlZGYzNzY1NWZiMWUzYTJmOWUyYWZpbHRlcnM9c3RhdHVzJTNBMQ==',
```
Comment (B77Mills) [inline sites/securityinfowatch.com/config/site.js:72]: @Jlaird Is this suppose to change? The one you are changing it to, I thought, was for the pre-ranked/sorted results.

File: packages/common/browser/algolia/site-search.vue [hunk 2]
```diff
@@ -3,6 +3,7 @@
     :index-name="tenantKey"
     :search-client="searchClient"
     :routing="routing"
+    :searchFunction="searchFunction"
```
Comment (B77Mills) [inline packages/common/browser/algolia/site-search.vue:6]: this needs to be :**search-function**="searchFunction"

Comment (Jlaird) [reply packages/common/browser/algolia/site-search.vue:6]: why?

Comment (Jlaird) [reply packages/common/browser/algolia/site-search.vue]: Done

**Review by B77Mills — APPROVED** [review]

**Pull request merged**